app/assets/javascripts/models/beta/iteration.js

```diff
@@ -182,13 +218,9 @@ export const groupBySprints = (stories = [], project, initialSprintNumber = 1, p
     const firstSprintIndex = 0;
     const isFromSprintInProgress = !Story.isUnstarted(story);
```

Comment:
I know this line wasn't created in your PR but I'm not sure I know what that means. This should be testing if this was already started somehow, right? I think it may be buggy because it will return true if the story is unscheduled. What do you think about creating a model method called `Story.isInProgress(story)` that check if it's neither unscheduled nor unstarted?

Reply:
Sure!